packages/shared/routes/setup/CustomNetwork.svelte

```diff
@@ -0,0 +1,55 @@
+<script lang="typescript">
```

Comment:
Please can we make the name of this component more descriptive so that we know it is an onboarding page.